main/src/com/google/refine/importers/tree/TreeImportingParserBase.java

```diff
@@ -210,8 +214,22 @@ protected void parseOneFile(
         boolean trimStrings = JSONUtilities.getBoolean(options, "trimStrings", true);
         boolean storeEmptyStrings = JSONUtilities.getBoolean(options, "storeEmptyStrings", false);
         boolean guessCellValueTypes = JSONUtilities.getBoolean(options, "guessCellValueTypes", true);
-
-        XmlImportUtilities.importTreeData(treeParser, project, recordPath, rootColumnGroup, limit2, trimStrings,
-                storeEmptyStrings,guessCellValueTypes);
+        
+        // copied from TabularImportingParserBase
```

Comment:
"copied from somewhere" always can be improved. will revisit the room to pull up to parent class.